core/domain/improvements_domain.py

```diff
@@ -50,9 +75,17 @@ class TaskEntry:
     """
 
     def __init__(
-            self, entity_type, entity_id, entity_version, task_type,
-            target_type, target_id, issue_description, status, resolver_id,
-            resolved_on):
+            self,
+            entity_type: str,
+            entity_id: str,
+            entity_version: int,
+            task_type: str,
+            target_type: str,
+            target_id: str,
+            issue_description: Optional[str],
```

Comment:
Why is this optional? In what circumstances would there be no description?

Reply:
The issue description is set as optional in storage.

https://github.com/oppia/oppia/blob/1f9fe1c065cdc6309d170c7165189b6d68c5bb39/core/storage/improvements/gae_models.py#L103
https://github.com/oppia/oppia/blob/1f9fe1c065cdc6309d170c7165189b6d68c5bb39/core/storage/improvements/gae_models.py#L273